src/main/java/mate/academy/StringBootIntro/model/Book.java

```diff
@@ -6,10 +6,12 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.Table;
 import jakarta.persistence.Id;
-import java.math.BigDecimal;
+import lombok.Data;
 
+import java.math.BigDecimal;
 @Entity
 @Table(name = "Books")
+@Data
```

Comment:
Don't use @Data on JPA entities, it can cause troubles, use @ Getter @ Setter separately

Reply:
Not fixed